bin/utils/processJSONFile.js

```diff
@@ -0,0 +1,73 @@
+const fs = require("fs");
+const path = require("path");
+const chalk = require("chalk");
+const { readFile } = require("jsonfile");
+const processFile = require("./processFile");
+const processFolder = require("./processFolder");
+const validateOutputFolder = require("./validateOutputFolder");
+
+const processJSONFile = (jsonFile) => {
+  readFile(jsonFile, (err, data) => {
+    if (err) {
+      console.error(chalk.red("Unable to read config file."));
+      return process.exit(1);
+    }
+
+    if (data.input === "") {
+      console.error(
+        chalk.red(
+          "Input file cannot be blank. Please specify an input file or folder."
+        )
+      );
+      return process.exit(1);
+    }
+
+    fs.lstat(data.input, (err, stats) => {
+      if (err) {
+        console.error(
+          chalk.red("Input file does not exist. Please use a different file.")
+        );
+        return process.exit(1);
+      }
+
```

Comment:
the `output` and `stylesheet` should be moved out of `ls.lstat()` so we could modularize this part